modules/@apostrophecms/ui/ui/apos/components/AposContextMenu.vue

```diff
@@ -141,15 +145,19 @@ const props = defineProps({
   dynamicFocus: {
     type: Boolean,
     default: false
+  },
+  keepOpenUnderModals: {
+    type: Boolean,
+    default: false
   }
 });
 
 const emit = defineEmits([ 'open', 'close', 'item-clicked' ]);
+const slots = useSlots();
 
 const isOpen = ref(false);
 const isRendered = ref(false);
 const placement = ref(props.menuPlacement);
-const event = ref(null);
```

Comment:
We pass event when existing directly to the emit. don't think we need this anymore.